src/core/core.js

```diff
@@ -416,13 +420,37 @@ export default class Core {
         this.updateHistoryAfterSearch(queryTrigger);
         window.performance.mark('yext.answers.universalQueryResponseRendered');
       }).catch(error => {
-        console.error('Universal search failed with the following error: ' + error);
-        this.storage.set(StorageKeys.UNIVERSAL_RESULTS, new UniversalResults({}));
-        this.storage.set(StorageKeys.DIRECT_ANSWER, new DirectAnswer());
-        this.storage.set(StorageKeys.LOCATION_BIAS, new LocationBias({}));
+        console.error('The following problem was encountered while processing universal search results: ' + error);
+        this._updateResultsSearchState(Searcher.UNIVERSAL, SearchStates.SEARCH_COMPLETE);
       });
   }
 
+  /**
+   * Update the search state of the results in storage
+   *
+   * @param {Searcher} searcherType
+   * @param {SearchStates} searchState
+   */
+  _updateResultsSearchState (searcherType, searchState) {
```

Comment:
This method is somewhat confusing. Part of that is probably because `SearchStates` isn't the greatest name. Since we always call this with `SearchStates.SEARCH_COMPLETE`, could we remove the second param and just call this `_markSearchComplete` or something?

Reply:
yup, will update